tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs

```diff
@@ -131,9 +131,45 @@ public static Result IsAspNetCoreMvcV1(this MockSpan span) => Result.FromSpan(sp
                 .Matches("component", "aspnet_core")
                 .Matches("span.kind", "server"));
 
-        public static Result IsAwsSqsV1(this MockSpan span) => Result.FromSpan(span)
+        public static Result IsAwsSqsInboundV1(this MockSpan span) => Result.FromSpan(span)
             .Properties(s => s
-                .Matches(Name, "sqs.request")
+                .Matches(Name, "aws.sqs.process")
+                .Matches(Type, "http"))
+            .Tags(s => s
+                .Matches("aws.agent", "dotnet-aws-sdk")
+                .IsPresent("aws.operation")
+                .IsOptional("aws.region")
+                .IsPresent("aws.requestId")
+                .Matches("aws.service", "SQS")
+                .IsPresent("aws.queue.name")
+                .IsOptional("aws.queue.url")
+                .IsPresent("http.method")
+                .IsPresent("http.status_code")
+                .IsPresent("http.url")
+                .Matches("component", "aws-sdk")
+                .Matches("span.kind", "consumer"));
+
+        public static Result IsAwsSqsOutboundV1(this MockSpan span) => Result.FromSpan(span)
+            .Properties(s => s
+                .Matches(Name, "aws.sqs.send")
+                .Matches(Type, "http"))
+            .Tags(s => s
+                .Matches("aws.agent", "dotnet-aws-sdk")
+                .IsPresent("aws.operation")
+                .IsOptional("aws.region")
+                .IsPresent("aws.requestId")
+                .Matches("aws.service", "SQS")
+                .IsPresent("aws.queue.name")
+                .IsOptional("aws.queue.url")
+                .IsPresent("http.method")
+                .IsPresent("http.status_code")
+                .IsPresent("http.url")
+                .Matches("component", "aws-sdk")
```

Comment:
From my comment [here](https://github.com/DataDog/dd-trace-dotnet/pull/4386/files#r1268761919), you'll need to add something like this to assert the peer.service tags for these spans and the other V1 SQS spans. Note: I also added peer.service.remapped_from which was likely another PR that was being done in parallel with your work.

```suggestion
                .IsPresent("peer.service")
                .IsOptional("peer.service.remapped_from")
                .MatchesOneOf("_dd.peer.service.source", "aws.queue.name", "peer.service")
                .Matches("component", "aws-sdk")
```

Reply:
Using suggestion in V1 span rules.